services/apps/alcs/src/alcs/search/notice-of-intent/notice-of-intent-search-view.entity.ts

```diff
@@ -92,6 +93,9 @@ export class NoticeOfIntentSubmissionSearchView {
   @ViewColumn()
   decisionDate: Date | null;
 
+  @ViewColumn()
+  legacyId?: string;
```

Comment:
Since you added a new ViewColumn, you'll need to use TypeORMs generate feature to create SQL to re-create the view with the new column.